pkg/cloud/powervs/assets/deployment.yaml

```diff
@@ -73,7 +73,6 @@ spec:
               source /etc/kubernetes/apiserver-url.env
             fi
             exec /bin/ibm-cloud-controller-manager \
-            --bind-address=127.0.0.1 \
```

Comment:
@hamzy, Trying to understand what made it to fail now or was it failing from long time? also we almost do same thing like IBMCloud platform, did you happen to check whats the status there as I could see the same flag being used there as well.

Reply:
@Karthik-K-N The cluster completes creation.  It does not pass CI testing.  I am not involved in IBMCloud platform development or testing.  This change only affects PowerVS.

Reply:
this has been failing most of the 4.16 cycle. @cjschaef has IBM Cloud seen this at all?

Reply:
Let's not hold up for that though.

Reply:
I would better suggest to know @cjschaef opinion. Since both PowerVS and IBMCloud use same CCM code, It would be better to be in sync.

Reply:
@Karthik-K-N How long are we going to block PowerVS CI on no response?

Reply:
We have experienced something similar during OCP Conformance (MonitorTests) related to liveness probes.
https://issues.redhat.com/browse/OCPBUGS-30267

The proposed change does not affect IBM Cloud VPC, since it is scoped to PowerVS, but I would be interested to see if this resolves the issue we have seen on 4.15+ with the MonitorTest results. Are those the same failures you mention with the CI?

Reply:
Yes, this should fix the MonitorTest failure.
@Karthik-K-N can we finally progress?!